### DISCUSSION TITLE
View Entries by Author in Control Panel

### DISCUSSION THREAD

**Discussion by angrybrad** [craftcms/cms#9462] [Ideas] [answered]
> Created by: Visiluna (pjones@visiluna.com) on 2015/06/16 22:04:13 +0000
> Votes at time of UserVoice import: 2

-----------------------------------

I frequently need to view all the entries by a particular author, but there doesn't appear to be any way to do this currently.

Would be great to put something like "author:name" in the entries search box. Or in the user's record, have a link to "view all entries by this author."

There are any number of ways this could be implemented. Any of them would be appreciated.

**Comment by janhenckens** [comment]
Bumping this one. In this case I have a search on the front-end of a site and searching for the author name doesn't return their own entries.

(solving this right now by adding a hidden field on the entry, where we copy the author name on save, so it does get added to the search index)

**Comment by MoritzLost** [comment]
Any current information for this topic? Finding all entries by a particular author seems to be a pretty standard requirement. The only approach I found is [this post on craftcms.SE](https://craftcms.stackexchange.com/a/33372/11705) which is at best a workaround. Would it be possible to add a support searching by author ID, username or email?

**Comment by brandonkelly** [comment, marked as answer]
This is currently under development for Craft 4.

**Reply by MoritzLost** [reply]
@brandonkelly Thats great to hear, thanks! Is there any publicly available roadmap / rough ETA for Craft 4 yet? I'm currently working on some sites that require features planned for Craft 4, so it would be good to know if its worth it to custom-build those feature or wait for them in Craft 4 ...

**Reply by brandonkelly** [reply]
We’re working on a public roadmap. Can’t give you an ETA on 4.0 just yet, sorry.

**Reply by brandonkelly** [reply]
We launched a public roadmap last week: https://craftcms.com/roadmap

**Reply by brandonkelly** [reply]
This has been implemented for Craft 4 🎉 (#10075)
